### ISSUE TITLE
Issues with npm start after updating to Mojave on Mac

### ISSUE THREAD

**Issue by saltnpixels** [ahmadawais/WPGulp#128]
**Is your feature request related to a problem? Please describe.**
When I run npm start I get an error:

``` Requiring external module @babel/register
[11:44:17] Local gulp not found in ~/Dropbox/projects/clients/MadDrone/mad
[11:44:17] Try running: npm install gulp
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your-project@1.0.0 start: `gulp`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the your-project@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/eric/.npm/_logs/2018-10-21T15_44_18_007Z-debug.log
```

I tried running install gulp but got nothing working.
Any ideas?

**Comment by saltnpixels** [comment]
**Fixed.**

Changed line in package.json from 
github:gulpjs/gulp#4.0

to

"gulp": "^4.0.0",

reran npm install.

Or just run:
npm install gulp@next

**Comment by lspoor** [comment]
I had a similar problem when trying to do npm install caused by `github:gulpjs/gulp#4.0` in my package.json, why is this used and not use `"gulp": "^4.0.0",`

**Comment by kevinhuddy** [comment]
I had problem setting up a project on a new computer this morning and it was caused by the same problem! Fixed by @saltnpixels message.

![screenshot 2018-10-23 12 58 33](https://user-images.githubusercontent.com/41078035/47377493-6b2edc00-d6c3-11e8-9c6b-5a83fac3584d.png)

**Comment by ahmadawais** [comment]
This should be fixed now in #129 otherwise open it again.

**Comment by parys-github** [comment]
Thanks, Ahmad.
Release 2.7.0 fixes all these frustrating problems with the installer :-)
Thanks and respect for you buddy.

**Comment by ahmadawais** [comment]
@parys-github glad to have helped. Just released v2.8 with another minor fix.

You know you can always [support my open source work →](https://github.com/ahmadawais/WPGulp#support-our-open-source-projects-)

Peace! ✌️
